src/index.js

```diff
@@ -1,21 +1,55 @@
-import ReactDOM from "react-dom";
-import React from "react";
+/**
+ * External dependencies
+ */
+import ReactDOM from 'react-dom';
+import React from 'react';
+import head from 'lodash/head';
 import { DropZoneProvider, DropZone } from '@wordpress/components';
 import { withState } from '@wordpress/compose';
+import apiFetch from '@wordpress/api-fetch';
+
+const validFileTypes = [ 'text/xml' ];
```

Comment:
Do we intend for `validFileTypes` to be extended in logic? We could wrap this array in `Object.freeze` to be sure of it's immutability.

```suggestion
const validFileTypes = Object.freeze( [ 'text/xml' ] );
```